hack/run-integration-tests.sh

```diff
@@ -47,4 +47,5 @@ fi
 KREW_BINARY="${krew_binary_realpath}"
 export KREW_BINARY
 
+-glog.v=4
```

Comment:
You sure you added it to the right place?

Reply:
I think. Maybe the other option is to add with the test command, something like this:

```
go test -test.v sigs.k8s.io/krew/integration_test v 4
```

Reply:
I feel like you aren’t sure :)
You just did what I described in the issue, so you just typed -glog.v=4 in the script on a new line, which won’t work. 
I don’t think your suggestion of adding `v 4`  to the go test as positional arguments  will work either.

Clearly the tests are also failing. Please test this locally by running the command.